src/isomorphic/classic/types/ReactPropTypes.js

```diff
@@ -85,6 +85,23 @@ var ReactPropTypes = {
   shape: createShapeTypeChecker,
 };
 
+/**
+ * inlined Object.is polyfill to avoid requiring consumers ship their own
+ * https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/is
+ */
+/*eslint-disable no-self-compare*/
+function is(x, y) {
```

Comment:
This seems like a weird place to define a polyfill.  Seems like we should do something more analogous to https://github.com/facebook/react/blob/master/src/shared/stubs/Object.assign.js